docs/slack-connector/installation.md

```diff
@@ -1,13 +1,63 @@
   - [Steps](#steps)
+  - [Verification](#verification)
+  - [Removal](#removal)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites-1)
+  - [Steps](#steps-1)
 
 ## Overview
 
-The Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
+The Slack Connector is a component which allows interaction with Slack API from inside of Kyma environment. The simplest way to install Slack Connector in Kyma is to install it as an Add-On.
+
+## Installation in Kyma as an Add-On
+
+### Prerequisites
+
+- Slack Bot with desired privileges installed to the destination workspace. See the tutorial provided by Slack on how to setup an application [here](https://api.slack.com/bot-users#getting-started). After you have created the Signing Secret, it appears in the application's **Settings** in the **Basic Information** section.
+- Access to Kyma Console
+
+> **OPTIONAL:** Follow these steps to install the default application. Be aware that it has **full permissions** in the in the workspace.
+>
+> 1. Go to the [authentication page](https://auth-slack.herokuapp.com/). Click the **Add to Slack** button, which redirects you to another page. Select the desired workspace and click **Allow**.
+>       - **NOTE:** If the link does not work, see [this](https://api.slack.com/docs/oauth#flow) tutorial in the Slack API documentation to create your own application.
+> 2. Copy the Bot Authentication Token and/or Slack Signing Secret. You will need it later in the installation process.
+
+### Steps
+
+1. In Kyma console, access the **Add-Ons Config** menu.
+2. Click **Add New Configuration** and fill in the **Urls\*** field with this URL:
+
+   ```http
+   github.com/kyma-incubator/hack-showcase//addons
+   ```
+
+3. Go to the Namespace in which you install the Connector.
+4. Find the Add-On in the Service Catalog and click it.
+5. Click **Add** and select the installation plan. Fill in all required fields and click **Create Instance**.
+6. Go to the **Services** tab in the Service Catalog. After provisioning and automatic registration of application's resources, the Service Class of Slack Connector appears here.
+7. Click the Service Class to enter its specification screen, click **Add once**, and then **Create Instance**.
+
+After you have created the service, you can easily bind it to the Lambda Function to allow the use of Slack Events.
+
+### Verification
+
+To verify correct configuration, check if Add-Ons and Service instances in the **Instances** area of the Service Catalog have status **RUNNING**.
+
+### Removal
+
+To correctly remove all Slack Connector resources, you must delete them in order reverse to the installation steps.
+> **NOTE:** Wait until deprovisioning and removing of all elements is complete before proceeding to next step to avoid possible errors. For example, after removing ServiceClass, the removal of ServiceInstance is impossible.
```

Comment:
```suggestion
> **NOTE:** Wait until deprovisioning and removing of all elements is complete before proceeding to the next step to avoid possible errors. For example, after removing ServiceClass, the removal of ServiceInstance is impossible.
```